vg/vgsvg/vgsvg.go

```diff
@@ -290,6 +291,8 @@ func large(a float64) int {
 	return 0
 }
 
+// FillString draws str at position pt using the specified font.
+// str is escaped with html.EscapeString.
```

Comment:
I'd prefer the wording I suggested for stylistic reasons.

Reply:
The function comment convention is to start the comment by the function name: `FillString ...`. 

Would we converge if I put this ?
``` Go
// FillString escape passed text with html.EscapeString.
```

Reply:
See here : https://blog.golang.org/godoc-documenting-go-code

Reply:
See https://github.com/gonum/plot/pull/512#pullrequestreview-195727529

```
// FillString draws str at position pt using the specified font.
// Text passed to FillString is escaped with html.EscapeString.
```